react/tic-tac-toe/src/app/screens/Game/components/MatchHistoryList/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import MatchesService from '../../../../../services/matchesService';
+import actionsCreators from '../../../../../redux/tictactoe/actions';
+import { parseMatchesResponse } from '../../../../../utils/mappers';
+
+import styles from './styles.module.scss';
+import MatchHistoryHeader from './components/MatchHistoryHeader';
+import MatchHistoryEntry from './components/MatchHistoryEntry';
+
+
+class MatchHistoryList extends Component {
+  componentDidMount() {
+    this.props.getHistoric();
+  }
+
+  render() {
+    const { historicEntries } = this.props;
+    return (
+      <div className={styles.matchContainer}>
+        <MatchHistoryHeader></MatchHistoryHeader>
+        {historicEntries.map(matchEntry => (
+          <MatchHistoryEntry key={matchEntry.date} matchEntry={matchEntry} c/>
+        ))}
+      </div>
+    );
+  }
+}
+
+MatchHistoryList.propTypes = {
+  getHistoric: PropTypes.func.isRequired,
+  historicEntries: PropTypes.arrayOf(PropTypes.object).isRequired
```

Comment:
avoid PropTypes.object. Instead use `PropTypes.shape` and specify the shape of the object: https://reactjs.org/docs/typechecking-with-proptypes.html